packages/main/src/Toast.js

```diff
@@ -0,0 +1,179 @@
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { getRTL } from "@ui5/webcomponents-base/dist/config/RTL.js";
+import ToastTemplate from "./generated/templates/ToastTemplate.lit.js";
+import ToastPlacement from "./types/ToastPlacement.js";
+
+// Styles
+import ToastCss from "./generated/themes/Toast.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-toast",
+	properties: /** @lends sap.ui.webcomponents.main.Toast.prototype */ {
+
+		/**
+		 * Defines the duration in milliseconds for which <code>ui5-toast</code>
+		 * remains on the screen before it's automatically closed.
+		 *
+		 * @type {number}
+		 * @defaultvalue 3000
+		 * @public
+		 */
+		duration: {
+			type: Integer,
+			defaultValue: 3000,
+		},
+
+		/**
+		 * Defines the placement of the <code>ui5-message-toast</code> web component.
+		 * <br><br>
+		 * Available options are:
+		 * <ul>
+		 * <li><code>TopStart</code></li>
+		 * <li><code>TopCenter</code></li>
+		 * <li><code>TopEnd</code></li>
+		 * <li><code>MiddleStart</code></li>
+		 * <li><code>MiddleCenter</code></li>
+		 * <li><code>MiddleEnd</code></li>
+		 * <li><code>BottomStart</code></li>
+		 * <li><code>BottomCenter</code></li>
+		 * <li><code>BottomEnd</code></li>
+		 * <ul>
+		 *
+		 * @type {ToastPlacement}
```

Comment:
string, instead of ToastPlacement (although it`s correct, we don`t have the enums documented as in openui5 demo kit, in separate pages, ideally this should be link to TaostPlacement page, but for now string is more useful for the users)